dev/com.ibm.ws.microprofile.mpjwt.1.1_fat_tck/fat/src/com/ibm/ws/microprofile/mpjwt11/tck/Mpjwt11TCKLauncher_aud_env.java

```diff
@@ -47,9 +49,6 @@ public static void tearDown() throws Exception {
     @AllowedFFDC("org.jose4j.jwt.consumer.InvalidJwtSignatureException")
     public void launchMpjwt11TCKLauncher_aud_env() throws Exception {
         String bucketAndTestName = this.getClass().getCanonicalName();
-        MvnUtils.setAdditionalMvnProps(new String[] { "-Dtck_appUndeployTimeout=60" }, server);
```

Comment:
`-Dtck_appUndeployTimeout=60` prop lost?

Reply:
that is the default value, didn't need to be set